pkgs/applications/misc/nerd-font-patcher/default.nix

```diff
@@ -1,36 +1,38 @@
-{ python3Packages, lib, fetchFromGitHub }:
+{ python3Packages, lib, unzip, fetchFromGitHub }:
 
 python3Packages.buildPythonApplication rec {
   pname = "nerd-font-patcher";
-  version = "2.2.2";
+  version = "3.0.0";
 
   # This uses a sparse checkout because the repo is >2GB without it
   src = fetchFromGitHub {
     owner = "ryanoasis";
     repo = "nerd-fonts";
     rev = "v${version}";
     sparseCheckout = [
-      "font-patcher"
-      "/src/glyphs"
+      "FontPatcher.zip"
```

Comment:
There is a separate release package. We can now avoid sparseCheckout und hand written unzip by using fetchzip with following path:
https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.1/FontPatcher.zip